src/components/Book.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { removeBook } from '../actions/index';
+
+const Book = props => {
+  const { book, deleteBook } = props;
```

Comment:
**OPTIONAL**: A pattern you may frequently find is to destructure on the argument.

<details>
 <summary>Hint</summary>
 
```suggestion
const Book = ({ book, deleteBook }) => {
```
</details>